libs/libevhtp/Makefile

```
@@ -44,6 +46,14 @@ define Package/libevhtp/description
    meaning the developer never thought of it being used for creating a full-fledged HTTP service.
 endef
 
+define Package/libevhtp/config
+config LIBEVHTP_STATIC_DEPENDS
+	tristate
+	default PACKAGE_libevhtp
```

```suggestion
	default PACKAGE_libevhtp

config LIBEVHTP_BUILD_DEPS
	tristate
	default m if LIBEVHTP_STATIC_DEPENDS
```

I suggest selecting the packages as `m`, so that libevent2-pthreads does not get included in the image if seafile-server (or even empty lbevhtp) is selected.

---

I think with this change `LIBEVHTP_STATIC_DEPENDS` can be a `bool` and I'll see if I can think of some better names than "static depends" and "build depends", but this is a good idea.